content/guides/accessibility/color-considerations.mdx

```diff
@@ -0,0 +1,144 @@
+
+In this use case, a user selecting this theme is likely intentionally expressing a desire for an experience with less visual contrast. Like other theme selections, this may be to satisfy aesthetic sensibilities, or access needs that are derived from visual conditions that make a higher contrast experience unusable. In the case of access needs, a higher contrast experience may actually be less accessible for the user.
+
+The availability of the Dark Dimmed theme treatment allows users to customize their GitHub experience without having to resort to browser extensions or other user-hosted modifications. Of note is the portability of this theme, in that it will automatically be available across multiple sessions and devices.
+
+While other sub-themes are not required to meet WCAG color contrast requirements, GitHub should strive to ensure they can be whenever possible. 
+
+As other sub-themes are not specifically designed to create a lessened contrast experience, there is less certainty that a user who is using the sub-theme has no contrast-related access needs. 
+
+Because of this, sub-themes such as Light/Dark Protanopia & Deuteranopia should strive to be WCAG color contrast compliant. This is especially relevant for Light/Dark High Contrast sub-themes.
+
+## Inputs
+
+Being able to perceive the boundaries of an input is vital. Meeting WCAG color contrast requirements helps to ensure the widest possible range of users can perform the data entry required to operate GitHub successfully.
+
+Broadly-speaking, there are four overall considerations for input contrast:
+
+1. Text,
+2. Borders, 
+3. Selected state contrast, and
+4. Icons.
+
+Combined, meeting contrast requirements allow a user to understand:
+
+1. What the input is for,
+2. What area of the input a user can interact with, and
+3. What state the input is in.
+
+If a computed color value in any of the four considerations is not compliant, it may endanger one or more portions of user understanding and operation of the input.
+
+### Text contrast
+
+Text portions of an input **must** meet a contrast ratio of `4.5:1` for "normal text", or `3:1` for "large text". This applies to both an input's label and its user-provided text content.
+
+The majority of GitHub input text size is "normal text". Both "normal text" and "large text" are WCAG-specific terms. More information about this can be found on [the Key Terms subsection of SC 1.4.3: Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html#dfn-large-scale).
+
+### Border contrast
+
+Borders communicate **the boundaries of an input**, and are considered non-text contrast. This includes inputs that allow for freeform user input such as text, and also pre-specified inputs such as checkboxes.
+
+Borders of an input **must** meet a contrast ratio of `3:1`. This is calculated by the border's computed color value compared to the computed background color the input is placed on.
```

Comment:
I understand this. I do want to mention that this is different from our original interpretation of the specs and will have a huge impact on githubs UI as basically all inputs are failing this atm.

The previous interpretation was: If the label meets `4.5:1` border does not need to.

Reply:
@lukasoppermann I'm thinking about putting together a feature flag test for this during passion week next week so we can just make a decision 😆 I think we just need to see it in action throughout GH.